scripts/create_test_data.R

```diff
@@ -36,7 +36,7 @@ pjnz_child <- testthat::test_path("testdata/bwa_aim-no-special-elig-numpmtct.PJN
 demp <- prepare_leapfrog_demp(pjnz_child)
 proj <- prepare_leapfrog_projp(pjnz_child)
 proj <- prepare_hc_leapfrog_projp(pjnz_child, proj)
-
+#proj$hc_age_coarse_cd4 <- as.numeric(proj$hc_age_coarse_cd4)
```

Comment:
What's the rational behind this line? It's an added line of code but commented out. Should something further be done with this in a future PR?  Maybe add a TODO comment here if so.

Reply:
Deleted.